api/controllers/game/spectate.js

```diff
@@ -0,0 +1,17 @@
+module.exports = async function (req, res) {
+  try {
+    const { gameId } = req.body;
+    const game = await gameService.populateGame({ gameId });
+
+    if (game.status || game.players.length !== 2) {
```

Comment:
```suggestion
    if (game.status || game.players.length < 2) {
```
It's probably much better to treat this like an actual number and use less then. Same effect but easier to understand imo. Unless you need it to be 2 exactly, in which case we can leave it. Wanted to pose the suggestion either way.

Reply:
I think that's reasonable